### PR TITLE
upgrade pulsar version to 2.8.0

### PR DESCRIPTION
**Pull request by Zhen-hao**

### CODE REVIEW

File: pulsar4s-avro/src/main/scala/com/sksamuel/pulsar4s/avro/package.scala [hunk 1]
```diff
@@ -59,7 +60,7 @@ package object avro {
     }
 
     override def getSchemaInfo: SchemaInfo = {
-      SchemaInfo.builder()
+      SchemaInfoImpl.builder()
```
Comment (sksamuel) [inline pulsar4s-avro/src/main/scala/com/sksamuel/pulsar4s/avro/package.scala:63]: Why the change to the impl ?

Comment (Zhen-hao) [reply pulsar4s-avro/src/main/scala/com/sksamuel/pulsar4s/avro/package.scala:63]: `builder` is no longer a method of `SchemaInfo` in 2.8.0. That's the most natural replacement I could find after reading the source code of Apache Pulsar.

Comment (sksamuel) [reply pulsar4s-avro/src/main/scala/com/sksamuel/pulsar4s/avro/package.scala:63]: Ok makes sense.
